lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -220,59 +239,95 @@ AppleObjCRuntimeV2 *SwiftLanguageRuntime::GetObjCRuntime() {
 }
 
 void SwiftLanguageRuntime::SetupSwiftError() {
-  Target &target(m_process->GetTarget());
-
-  if (m_SwiftNativeNSErrorISA.hasValue())
-    return;
-
-  ConstString g_SwiftNativeNSError("__SwiftNativeNSError");
+  m_SwiftNativeNSErrorISA =
+      FindSymbolForSwiftObject(m_process->GetTarget(), RuntimeKind::Swift,
+                               "__SwiftNativeNSError", eSymbolTypeObjCClass);
+  m_initialized_swift_native_error_isa = true;
```

Comment:
We assert that this is true in the function below, but we set it unconditionally here. Should it be set conditionally on `m_SwiftNativeNSErrorISA != LLDB_INVALID_ADDRESS`?

Reply:
Same as above, this signals that the initialization has been run, not whether it was successful.